README.rst

```diff
@@ -23,6 +21,37 @@ machine code.
 For more information, see `the documentation
 <https://cretonne.readthedocs.io/en/latest/?badge=latest>`_.
 
+Status
+------
+
+Cretonne currently supports enough functionality to run a wide variety of
+programs, including all the functionality needed to execute WebAssembly MVP
+functions, although it needs to be used within an external WebAssembly
+embedding to be part of a complete WebAssembly implementation.
+
+The x86-64 backend is currently the most complete and stable; other
+architectures are in various stages of development. Cretonne currently supports
+the System V AMD64 ABI calling convention used on many platforms, but does not
+yet support the Windows x64 calling convention. The performance of code
+produced by Cretonne is not yet impressive, though we have plans to fix that.
+
+The core codegen creates have minimal dependencies, and do not require any host
+floating-point support. Support for `no_std` mode in the core codegen crates is
+[in development](https://github.com/Cretonne/cretonne/tree/no_std).
```

Comment:
Is this a correct link format?

Reply:
Oops, I used markdown format, and this is a reST file. Thanks for spotting that! I'll fix it.